.prout.json

```diff
@@ -0,0 +1,5 @@
+{
+  "checkpoints": {
+    "DEV": { "url": "http://wrangler-todo.herokuapp.com", "overdue": "10M" },
```

Comment:
Slightly out of the scope of your application but was wondering wether you could suggest how to retrieve the sha of HEAD from a heroku app.  

-- 
graham hadgraft
Sent with Airmail

On 21 October 2015 at 13:10:44, Roberto Tyley (notifications@github.com) wrote:

In .prout.json:

> @@ -0,0 +1,5 @@
> +{
> +  "checkpoints": {
> +    "DEV": { "url": "http://wrangler-todo.herokuapp.com", "overdue": "10M" },
That great big exception is caused by a missing APPLICATION_SECRET environment variable, which you need to configure in Heroku. Apologies for this, I've only just added that required field to the Deploy to Heroku button - so you can manually add that field to your Heroku app:



...or deploy a new one.

—
Reply to this email directly or view it on GitHub.